src/bson.rs

```diff
@@ -801,23 +770,29 @@ impl Bson {
     }
 }
 
-/// `TimeStamp` representation in struct for serde serialization
-///
-/// Just a helper for convenience
-///
-/// ```rust,ignore
-/// use serde::{Serialize, Deserialize};
-/// use bson::TimeStamp;
-///
-/// #[derive(Serialize, Deserialize)]
-/// struct Foo {
-///     timestamp: TimeStamp,
-/// }
-/// ```
+/// Represents a BSON timestamp value.
 #[derive(Debug, Eq, PartialEq, Clone, Copy, Hash)]
 pub struct TimeStamp {
```

Comment:
This struct already existed, so I just reused it and changed the field names to be more descriptive.